kubectl-fdb/cmd/restart.go

```diff
@@ -84,36 +75,29 @@ func newRestartCmd(streams genericclioptions.IOStreams) *cobra.Command {
 				return err
 			}
 
-			namespace, err := getNamespace(*o.configFlags.Namespace)
-			if err != nil {
-				return err
-			}
-
-			cluster, err := loadCluster(kubeClient, namespace, clusterName)
+			processGroupsByCluster, err := getProcessGroupsByCluster(cmd, kubeClient, processGroupSelectionOpts)
 			if err != nil {
 				return err
 			}
-
-			var processes []string
-			if allProcesses {
-				pods, err := getPodsForCluster(kubeClient, cluster)
-				if err != nil {
-					return err
-				}
-
-				for _, pod := range pods.Items {
-					processes = append(processes, pod.Name)
+			for cluster, processGroupIDs := range processGroupsByCluster {
+				// TODO remove this once we've removed support for processGroupID (instead of pod) lookup and
+				//  can more nicely convert getProcessGroupsByCluster to return podNames (possible now, but less clean)
```

Comment:
would be good to find the issue, though I will probably just make it directly get the podNames tomorrow

Reply:
Hm, I'm pretty sure there should be an issue for it but I'm not able to find it right now 🤔